load/Validate.py

```diff
@@ -76,6 +77,33 @@ def process(self, filesets):
 		#duplicates = find.findDuplicates()
 		#find.moveDuplicates(duplicates)
 
+	def validateTextPlainFilesets(self, texts, inp, filePath):
+		errorTuple = texts.validateFileset("text_plain", inp.bibleId, inp.filesetId, inp.languageRecord, inp.index, filePath)
+
+		if errorTuple == None:
+			errorTuple = texts.invokeBiblePublisher(inp, filePath)
+
+			if errorTuple == None:
+				return texts.createTextFileset(inp)
+
+		logger = Log.getLogger(inp.filesetId)
+		logger.messageTuple(errorTuple)
+
+		return None
+
+	def validateTextJsonFilesets(self, texts, inp, filePath):
```

Comment:
@bradflood I have created this new method to invoke validateFileset method for a `text_json` fileset and to invoke new invokeSofriaCli method.